### ISSUE TITLE
TAG appointment should be via IETF style nomcom

### ISSUE THREAD

**Issue by nrooney** [w3c/w3process#230]
The TAG is currently elected. We could move to an IETF-style NomCom appointment. This ensures the right people for the role are selected and a balanced TAG can be achieved. Comments are welcome.

- The NomCom consists of a random set of volunteers who meet a set criteria - e.g. have chaired a group or published a draft in the last X years. The AB could be members of the NomCom. The NomCom should have advising members. They cannot vote, only advise. The TAG Chair, a Developer Invited Expert would be good choices.
- Nominations for groups go direct to the NomCom. Nominations can come from any source. The NomCom will see via the application what the source of the nomination is - the NomCom can choose to weigh an AC Nomination in a positive manner if they wish. This means that AC nominations count for something, but other groups (e.g. independent developers) are not blocked from applying.
- NomCom sees nominations as they come in. If they feel a diverse pool isn’t being reached (diversity for all items including gender, technical knowledge of particular topics, organisation representation size, global location, etc.) the NomCom will begin pushing for more candidates.
- NomCom starts the process of selecting candidates. This is as follows:
  - The NomCom meets before evaluating the candidates. It agrees on key qualities it is looking for in the candidates for the open roles. It is likely these qualities will be the same in each election, although sometimes they will be different (e.g. “Chair of the TAG" will require different qualities to “TAG Member”). Consensus on these qualities is reached.
  - All applications are read. Obvious “no” candidates are discarded.
  - The NomCom meets to discuss the “possible” candidates remaining. Further candidates are discarded for reasons the NomCom deem appropriate. These could be because there has been an influx of candidates which represent one industry, or many candidates are experts in only one field.
  - The NomCom is left with the final set of “possible candidates”. These candidates are interviewed for the role at TPAC, at the AC Meeting or via teleconference. Video Conference interviews are acceptable.
- The NomCom announces the selected candidates. An objection process opens to allow formal objections from the AC. Formal objections will be handled as normal. After 28 days, the formal objection window closes and the newly selected individuals begin their term.

**Comment by michaelchampion** [comment]
I will drop my objection to removing the “one vote per open seat” language that some consider a bug if something like this proposal is adopted.

**Comment by dwsinger** [comment]
@michaelchampion I think what is meant by "approval ballot" is that AC is asked a single question:
* do you approve the slate as presented? [yes/no/abstain]

The NomCom presents an exact slate. The vote is not by individual. Correct me if I am wrong.

**Comment by michaelchampion** [comment]
I'm not sure.  https://en.wikipedia.org/wiki/Approval_voting  says:

> Approval voting ballots show a list of the candidates running for that seat for each office being contested. Next to each name is a checkbox (or another similar way to mark "Yes" or "No" for that candidate).

> Each candidate may be treated as a separate question: "Do you approve of this person for the job?". Approval voting lets each voter indicate support for one, some, or all candidates. All votes count equally, and everyone gets the same number of votes: one vote per candidate, either for or against. 

That's more or less what I thought Approval Voting was.  But then the Wikipedia article says:

> Final tallies show how many voters support each candidate, and the winner is the candidate whom the most voters support.

That's a bit different than either of our summaries.    I was thinking that there could be multiple winners, those that reach some threshold like getting support form 50% (or maybe 67% or 75%) of the electorate.  Maybe that's not a well-understood voting system ... but I'm just thinking about tweaks, not proposing an alternative to what @nrooney wrote at this point.

**Comment by chaals** [comment]
My biggest concern with this is how we select the NomCom, and how the NomCom itself decides on candidates. Overall I am not a big fan of the proposal, but I would at least like to see details about NomCom selection and decision-making before I would consider such a proposal.

**Comment by chaals** [comment]
@dwsinger it seems you and @michaelchampion are talking about different things.

**IFF** we have an approach like this, I think it makes more sense to approve or reject the slate in its entirety than intropduce yet anohter voting system. In the "approval voting" scenario @michaelchampion proposes, what happens when the AC rejects the "accessibility" and "privacy" candidates from the TAG, who happen to be the ones that bring the most developer interaction and expertise in performance? Can the NomCom reopen the whole slate in order to get a different balance? Do they have to present new candidates? What if those are rejected too...?

**Comment by michaelchampion** [comment]
I don't necessarily have a preferred approach here.  I am strongly in favor of the basic approach -- a group of people -- with lots of knowledge/experience about how W3C and TAG works and appreciate the various qualities the TAG as a whole needs to have -- selects a slate of qualified people, then the AC has some mechanism to confirm or object to that slate.  My initial concern about the "NomCom selects, AC can object" is that it wasn't clear to me how objections would get processed. I wondered if an "approval ballot" was a way for the AC to do the confirmation; Dave pressed on whether that means approving the *slate* or approving each *individual.*  I admitted that the term "approval ballot" is not clearly defined and we should discuss.

After a bit more thought, the principles we need to consider our choice of a confirmation/objection mechanism would seem to be:
- Minimal friction in the hopefully typical case where the NomCom does a satisfactory job
- An ability for the membership force a reset of some sort if there is a widespread sentiment that the NomCom made bad choices
- Some fairly deterministic mechanism to handle the reset

Reset option 1: If we have an engaged Director whose decisions will be respected by the membership, the way @nrooney implies it in the issue would probably work:  AC reps can object to specific selections by the NomCom, the Director considers and either approves the NomCom nominee or appoints someone else who in the Director's judgment brings the range of skills/background needed to complement the rest of the TAG. If the community believes this describes our current situation well enough to approve this Process change, I'm OK with that.

Reset option 2:  I'm a bit skeptical that the current setup where W3M makes the vast majority of the decisions formally made by the (disengaged) Director is suitable to the task.  My rough thinking about an approval ballot was that there would be a flexible number of seats on the TAG, and any people selected by the NomCom but who didn't get (maybe) 50% voting Yes in a ballot just wouldn't be seated.  I suppose we could have a special TAG election under the current system for any seats whose NomCom selections didn't get AC approval, so such seats wouldn't stay vacant for a whole year.

Reset option 3: Or we could do what Dave's question implies and Chaals seems to endorse:  If the AC doesn't approve the whole slate selected by the NomCom, the reset mechanism would choose the entire set of seats whose term is expiring. For example, maybe we could just re-do the TAG election using the current system. That seems more disruptive than the other alternatives I've listed, but perhaps the fear of that disruption will motivate the NomCom to do their work well enough to ensure it never happens. 

I'm not really sure about my preference order for 1,2,3 above, and I prefer all of them to the current system.     I hope others can add options or refine these ideas.

**Comment by dwsinger** [comment]
Right, I understand IETF-style NomCom as that the NomCom presents a slate of exactly the correct number of candidates, and that entire slate is then ratified (or not) in a simple single yes/no vote.

Which, as chaals says, moves the whole discussion back to how one chooses the NomCom in the first place.

**Comment by michaelchampion** [comment]
A strawman proposal for selecting the TAG NomCom:

- For every TAG election, Identify a pool of NomCom members candidates, probably a) the AB, b) TAG members who are not up for re-election, c) chairs/editors with long experience, d) maybe a few others identified by W3M.
- Ask each member of that pool whether or not they would make a commitment to do the work required of a NomCom member, remove those who won't/can't from the pool.
- Randomly select 5 people from the pool (maybe with some constraints, e.g. at least 1 from the AB, at least 1 from the TAG, don't select anyone who was on the last NomCom)

Someone designated by W3M might serve as "mentor" or whatever to ensure each NomCom understands what's expected, what worked in the past, etc.

**Comment by frivoal** [comment]
(just for the record, this issue isn't being ignored, merely handled as part of Process 2020, since it is not an objection against Process 2019)

**Comment by frivoal** [comment]
A draft proposal loosely based on the the various times we had that discussion is included in the "director-free" experimental branch of the process: https://w3c.github.io/w3process/director-free/#TAG-appointments ([diff version](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fw3process%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fw3process%2Fdirector-free%2F#TAG-appointments))

**Comment by frivoal** [comment]
The AB (and TAG) resolved to do a NOMCOM, and after years of exploration, decided it was too complicated for our use case, and resolved to do what is now documented at https://www.w3.org/Consortium/Process/Drafts/#TAG-appointments instead.

**Comment by frivoal** [comment]
@nrooney It's a little strange asking after such a long time, but (for disposition of comments purposes), I'd like to ask if you can confirm that you accept the conclusion here.

This issue is kind of both accepted and rejected at the same time. We did agree with you, and did draft a NOMCOM based process for (part of) the TAG, so in that sense, this issue is closed as accepted. But also, after working on that and refining it for a long time (years), we walked it back: see https://github.com/w3c/w3process/pull/611#issuecomment-1196692082 and https://www.w3.org/2022/08/16-ab-minutes, and we ended up reverting this NOMCOM in favor of the Team doing the nomination and the TAG doing the ratification of the nominees, as described in https://www.w3.org/Consortium/Process/Drafts/#TAG-appointments

**Comment by mnot** [comment]
It's really hard to track the decision here; you refer to a PR, nto an issue. Is there an _issue_ tracking this decision?

**Comment by frivoal** [comment]
It is a little hard to follow.
I believe the source of the discussion came from multiple places, notably the [PR as I mentioned](https://github.com/w3c/w3process/pull/611#issuecomment-1196692082), as well as informal conversations with the TAG IIRC. Then the decision to switch away from a NomCom to the now current approach was [made in an AB meeting](https://www.w3.org/2022/08/16-ab-minutes#r02) (the one whose [minutes](https://lists.w3.org/Archives/Member/w3c-ac-members/2022JulSep/0046.html) I linked to), and that decision was implemented in the same PR, with review by, and then with the assent of the CG after several calls for review (in [this agenda](https://www.w3.org/mid/1dff157e-d613-fd14-b37a-44de6af6eece@inkedblade.net), in [this call for review](https://github.com/w3c/w3process/pull/611#issuecomment-1227709567), in [that other agenda](https://www.w3.org/mid/a12e80dd-c919-bd89-727f-f3ac4e0f4efb@w3.org)).

**Comment by nrooney** [comment]
Thanks for asking me to respond to this. This is acceptable as an interim approach, especially as there are a number of moving parts at W3C currently and focus is likely best placed in other areas. Returning to this at a later date once prioritised items are settled would be encouraged.
